orchestrator/assets/waste.py

```diff
@@ -113,6 +119,45 @@ def small_stream_recycle(dhub: ResourceParam[DataHubResource]):
     return df_out
 
 
+@asset(
+    io_manager_key="postgres_replace",
+    compute_kind="python",
+    group_name="raw",
+)
+def waste_emission_factors_EPA(dhub: ResourceParam[DataHubResource]):
+    """This asset ingest the Commuting Emission Factors by vehicle
+    type from EPA Emission Factors Hub."""
+    project_id = dhub.get_project_id("Scope3 General")
+    logger.info(f"Found project id: {project_id}!")
+    download_links = dhub.search_files_from_project(project_id, "EPA_GHG_emission_factors_June_2024")
+    if len(download_links) == 0:
+        logger.error("No download links found!")
+        return pd.DataFrame()
+    # Load the data
+    workbook = pd.ExcelFile(download_links[0], engine="openpyxl")
+
+    # Get GWP to calculate CO2-eq
```

Comment:
gets the emission factors data from the Excel file

Reply:
nice catch!